doc/source/cluster/kubernetes/examples/gpu-training-example.md

```diff
@@ -43,7 +43,7 @@ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container
 # Install both CRDs and KubeRay operator.
 helm repo add kuberay https://ray-project.github.io/kuberay-helm/
 helm repo update
-helm install kuberay-operator kuberay/kuberay-operator --version 1.2.2
+helm install kuberay-operator kuberay/kuberay-operator --version 1.3.0
```

Comment:
I will rewrite this doc after the release. Track the progress https://github.com/ray-project/kuberay/issues/3049